recipes/djinni-support-lib/all/test_package/test_package.cpp

```diff
@@ -6,6 +6,3 @@
 int main() {
     return EXIT_SUCCESS ;
 }
-
```

Comment:
In general, imho, please do not make such changes. It adds no better functionality, it's just about personal taste 
However, if you want to have that change,  please let at least 1 empty line is at the end.